cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -558,6 +539,18 @@ storage:
             volumeMounts:
             - name: config-volume
               mountPath: /etc/nginx
+          - name: etcd-proxy
+            image: registry.opensource.zalan.do/teapot/etcd-proxy:master-2
+            - {{ .Cluster.ConfigItems.etcd_endpoints }}
+            ports:
+            - containerPort: 2379
+            resources:
+              requests:
+                cpu: 25m
+                memory: 25Mi
+              limits:
```

Comment:
I would not have any explicit limits here. We don't particularly care about them on the master nodes, and I we don't want to debug OOMs.